src/components/AssetAccountDetails.tsx

```diff
@@ -38,11 +41,13 @@ export const AssetAccountDetails = ({ assetId: caip19, accountId }: AssetDetails
           <UnderlyingToken assetId={caip19} accountId={accountId} />
           <AssetTransactionHistory limit={3} assetId={caip19} accountId={accountId} />
         </Stack>
-        <Stack flex='1 1 0%' width='full' maxWidth={{ base: 'full', xl: 'sm' }} spacing={4}>
-          <TradeCard defaultBuyAssetId={caip19} />
-          <AssetMarketData assetId={caip19} />
-          <AssetDescription assetId={caip19} />
-        </Stack>
+        {!assetUnavailable && (
+          <Stack flex='1 1 0%' width='full' maxWidth={{ base: 'full', xl: 'sm' }} spacing={4}>
+            <TradeCard defaultBuyAssetId={caip19} />
+            <AssetMarketData assetId={caip19} />
+            <AssetDescription assetId={caip19} />
```

Comment:
we don't want to hide the description unnecessarily - it's just market data that's unavailable, not necessarily asset descriptions